arrocke/gloss-translation
# feat: configure text direction on languages

## What has changed

* add button selector input component for radio buttons
* added language setting for text direction
* modify the text direction of the gloss selector based on the language setting



## Connected Issues



closes #223 

## QA Steps



1. Change the text direction to RTL
2. Try typing in the gloss selector and confirm that text direction is what is expected
3. Change the text direction back to LTR
4. Try typing again

## Post-Deployment
